lib/shared/screens/admin/shared/components/content-page-builder/index.js

```diff
@@ -90,6 +90,12 @@ export default class ContentPageBuilderContainer extends Component {
     confirmUnpublish: PropTypes.bool.isRequired
   };
 
+  static defaultProps = {
+    publish: () => {},
+    unpublish: () => {},
```

Comment:
I removed the default proptypes for these func proptypes.  I also removed the isRequired flag and there don't seem to be any issues. Updated pull request.